File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/CropSpecificInfo/CropSpecificInfoCreateDto.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace OregonTilth.Models.DataTransferObjects
{
    public class CropSpecificInfoCreateDto
    {
        public int WorkbookID { get; set; }
        public int CropID { get; set; }
        public CropSummaryDto Crop { get; set; }
        public int TpOrDsTypeID { get; set; }
        public TpOrDsTypeDto TpOrDsType { get; set; }
        public int RowsPerStandardWidth { get; set; }
        public int DripTapeRowsPerStandardWidth { get; set; }
        public int? InRowSpacing { get; set; }
        public decimal? SeedCostPerStandardUnitOfSpace { get; set; }
        public decimal? TransplantProductionCostOutsourced { get; set; }

    }
}

File: Source/OregonTilth.Web/src/app/shared/models/forms/crop-specific-info/crop-specific-info-create-dto.ts
import { CropDto } from "../../generated/crop-dto";
import { TpOrDsTypeDto } from "../../generated/tp-or-ds-type-dto";

export class CropSpecificInfoCreateDto {
    WorkbookID: number;
    CropID: number;
    Crop: CropDto;
    TpOrDsTypeID: number;
    TpOrDsType: TpOrDsTypeDto;
    RowsPerStandardWidth: number;
    DropTapeRowsPerStandardWidth: number;
    InRowSpacing: number;
    SeedCostPerStandardUnitOfSpace: number;
    TransplantProductionCostOutsourced: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}
